src/components/MissionArea/MissionArea.js

```diff
@@ -0,0 +1,12 @@
+import './MissionArea.css';
+
+const MissionArea = () => {
```

Comment:
I think we'll have to change the story names and mission names around or change the text but it looks good!